lib/predictions/repo.ex

```diff
@@ -59,18 +62,24 @@ defmodule Predictions.Repo do
 
   @spec filter_predictions([Parser.record()] | {:error, any}, DateTime.t() | nil) ::
           [Parser.record()] | {:error, any}
-  defp filter_predictions(predictions, min_time \\ nil)
+  defp filter_predictions(predictions, min_time \\ nil, include_terminals \\ false)
 
-  defp filter_predictions({:error, error}, _) do
+  defp filter_predictions({:error, error}, _, _) do
     {:error, error}
   end
 
-  defp filter_predictions(predictions, min_time) do
+  defp filter_predictions(predictions, min_time, _include_terminals = false) do
```

Comment:
This is an odd use of the match operator, and is unnecessary for pattern matching in the function head.
```suggestion
  defp filter_predictions(predictions, min_time, false) do
```

Reply:
How about [this](https://github.com/mbta/dotcom/pull/2825/commits/770157c93c1432c7cb80faf888c66ff3b3cfc0e6)?

Can't pattern-match on `Keyword` args, but can do boolean logic on them!